scripts/pr-show-commits

```diff
@@ -0,0 +1,21 @@
+# https://cli.github.com/
```

Comment:
```suggestion
#!/usr/bin/env sh

# https://cli.github.com/
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1466/commits/e3545f0c0930d4401583e973b2c8b26f7639b439